### PR TITLE
Draft ietf lpwan schc over lorawan 06

### PR DESCRIPTION
**Pull request by Oliv4945**
- Add IID collision
- Fix typos
- Update security
- Improve SCHC ACK following Arun comment
- Fix RFC compilation
- Clarify last tile

### CODE REVIEW

File: draft-ietf-lpwan-schc-over-lorawan.md [hunk 1]
```diff
@@ -426,9 +427,12 @@ Exemple with:
 2. cmac: 0x4E822D9775B2649928F82066AF804FEC
 3. IID: 0x28F82066AF804FEC
 ~~~~
-
 {: #Fig-iid-computation-example title='Example of IID computation.'}
 
+There is a small probability of IID collision in a network, if such event occurs
+the IID can be changed by rekeying the device on L2 level (ie: LoRaWAN rejoin).
```
Comment (jca-klk) [inline draft-ietf-lpwan-schc-over-lorawan.md]: I would not say "rejoin" as it has a specific meaning in LoRaWAN 1.1.x. 
"LoRaWAN new join" ?

File: draft-ietf-lpwan-schc-over-lorawan.md [hunk 2]
```diff
@@ -739,9 +751,12 @@ RECOMMENDED value is 12 hours for both Class B and Class C end-devices.
 
 This document is only providing parameters that are expected to be better
 suited for LoRaWAN networks for {{I-D.ietf-lpwan-ipv6-static-context-hc}}. IID
-security is discussed in {{IID}}.As such, this document does not contribute to
+security is discussed in {{IID}}. As such, this document does not contribute to
 any new security issues in addition to those identified in
 {{I-D.ietf-lpwan-ipv6-static-context-hc}}.
+Moreover SCHC data (LoRaWAN payload) are protected on LoRaWAN level by an AES-128
+encryption with key shared by device and SCHC gateway. Those keys are renew each
+LoRaWAN session (ie: each join or rejoin to the network)
```
Comment (jca-klk) [inline draft-ietf-lpwan-schc-over-lorawan.md:760]: Here, "rejoin" has the LoRaWAN 1.1.x meaning. Keep it.

**Review by jca-klk — APPROVED** [review]
OK for me, except "new join" vs "rejoin" comment 👍

File: draft-ietf-lpwan-schc-over-lorawan.md (hunk 1 above)
Comment (Oliv4945) [reply draft-ietf-lpwan-schc-over-lorawan.md]: What about `(ie: trigger a LoRaWAN join)` instead ?

Comment (jca-klk) [reply draft-ietf-lpwan-schc-over-lorawan.md]: > What about `(ie: trigger a LoRaWAN join)` instead ?

Perfect!

**Review by ivajloip — APPROVED** [review]
Looks good to me :slightly_smiling_face:

**Pull request merged**